lib/node_modules/@stdlib/math/base/special/besselj0/README.md

```diff
@@ -0,0 +1,62 @@
+# besselj0
+
+> Compute the [Bessel function of the first kind][bessel] of order zero.
+
+
+<section class="usage">
+
+## Usage
+
+``` javascript
+var besselj0 = require( '@stdlib/math/base/special/besselj0' );
+```
+
+#### besselj0( x )
+
+Compute the [Bessel function of the first kind][bessel] of order zero at `x`.
+
+``` javascript
+var v = besselj0( 0.0 );
+// returns 1.0
+
+v = besselj0( 1.0 );
+// returns 0.76519768655796649
+
+v = besselj0( NaN );
+// returns NaN
+```
+
+</section>
+
+<!-- /.usage -->
+
+
+<section class="examples">
+
+## Examples
+
+``` javascript
+var randu = require( '@stdlib/math/base/random/randu' );
+var besselj0 = require( '@stdlib/math/base/special/besselj0' );
+
+var x;
+var i;
+
+for ( i = 0; i < 100; i++ ) {
+    x = randu() * 10.0;
+    console.log( 'besselj0(%d) = %d', x, besselj0( x ) );
+}
+```
+
+</section>
+
+<!-- /.examples -->
+
+
+<section class="links">
+
+[bessel]: https://en.wikipedia.org/wiki/Bessel_function#Bessel_functions_of_the_first_kind
```

Comment:
Should add the link to the link database (`make links-insert`) and assign a unique id. I believe, here, something akin to `bessel-j0`. If you want to be more general, use the current id, but link to the general Wikipedia page, not a subsection.